Makefile

```diff
@@ -418,3 +410,21 @@ help:
 print-%:
 	@echo '$*=$($*)'
 
+#>>>
+# run integration tests
+# make build-all -P
+
+#<<<
+.PHONY: build-all
+build-all:
+	./pre-build-check.sh
+
+# Setup the entire stack
+	$(MAKE) init-docker
+	$(MAKE) build-minio
+	$(MAKE) build-htsget-server
+	$(MAKE) build-chord-metadata
```

Comment:
You shouldn't have to individually build modules: if you run make compose, they should get pulled or built as needed, based on the CANDIG_MODULES variable in .env.

Reply:
If you want, you can add the make docker-pull target here instead, so that you get some version pulled down.